lib/ledger_sync/ledgers/netsuite/resources/customer_depost.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
```

Comment:
This file name should be `customer_deposit.rb` (missing the `i`).

Reply:
Great catch!